project/project.go

```diff
@@ -19,6 +20,8 @@ var (
 	file ini.File
 
 	pp = ""
+
+	copyToRoot = flag.Bool("r", false, "set root of the project as build out. should not be used with watch")
```

Comment:
I think it should be impossible for users to create that kind of error. They will blame you every time they make that mistake.